LICENSE.txt

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2016 OSS Friday
+Copyright (c) 2016 Open Source Friday
```

Comment:
Thanks! Removing the year entirely because they aren't legally required and require infinite updating 😆